src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menDiedSelectCentury = people.filter(human => {
+    const sexIsMale = human.sex === 'm';
+    const diedInSelectCentury = century
+      ? Math.ceil(human.died / 100) === century
+      : human.born;
+
+    return (sexIsMale && diedInSelectCentury);
+  });
```

Comment:
let's do something like this:

```
const men = getBySex(...);
const menDiedInSelectedCentury = men.filter(...);
```
by doing so, you make sure you split tasks logically so that it doesn't look messy